cpp/src/arrow/memory_pool.h

```diff
@@ -71,7 +71,7 @@ class ARROW_EXPORT MemoryPool {
 class ARROW_EXPORT LoggingMemoryPool : public MemoryPool {
  public:
   explicit LoggingMemoryPool(MemoryPool* pool);
-  virtual ~LoggingMemoryPool() = default;
+  ~LoggingMemoryPool() override = default;
```

Comment:
Unless they are base classes of course :)

Reply:
If they are base classes, they should not need override.

Reply:
I meant that you can't remove virtual destructors from base classes.